doc/api/https.md

```diff
@@ -124,12 +124,13 @@ added: v0.3.4
 
 See [`http.Server`][] for more information.
 
-### `server.close([callback])`
+### `server.close([options][, callback])`
```

Comment:
I think this is not correct. `https.Server` does not inherit from `http.Server`.

Reply:
Thanks for noticing. I think you are right. Once we agree on API changes I'll add the same in `https.Server`.